gpkit/constraints/sgp.py

```diff
@@ -184,44 +182,38 @@ def firstgp(self, x0, substitutions):
                 else:
                     self.is_sgp = True
```

Comment:
punting the creation of specific `SignomialProgram` and `signomialsolve` methods to another issue